docs/content/en/docs/implementing/integrate.md

```diff
@@ -102,6 +103,12 @@ These keys are defined as:
    of the generated `KeptnApp` representing your Application.
    All Workloads that share the same value for this label
    are consolidated into the same `KeptnApp` resource.
+* `keptn.sh/container`: Determines the name of the container in the Workload,
+   from which Keptn extracts the version.
+   This applies to single- and multi-container
+   workloads.
+   If the given container name does not match any container in the workload
+   no version can be determined.
```

Comment:
you could add a note here that there's no `app.kubernetes.io/` equivalent for the `/container` label.
```suggestion
   If the given container name does not match any container in the workload
   no version can be determined.
   Note that there is no equivalent `app.kubernetes.io/` annotation/label for this label.
```